compiler/rustc_infer/src/infer/mod.rs

```diff
@@ -527,9 +527,6 @@ pub enum RegionVariableOrigin {
     Coercion(Span),
 
     /// Region variables created as the values for early-bound regions.
-    ///
-    /// FIXME(@lcnr): This can also store a `DefId`, similar to
-    /// `TypeVariableOriginKind::TypeParameterDefinition`.
```

Comment:
THis one still holds, the `DefId` has been moved to https://doc.rust-lang.org/nightly/nightly-rustc/rustc_infer/infer/type_variable/struct.TypeVariableOrigin.html#structfield.param_def_id

Reply:
I will update the comment, but I still find it confusing and would like to clarify it. Is "can" the wrong word? Should it be replaced by "could" or "should"? I don't understand what the FIXME comment is saying.

Reply:
it should use a `DefId` imo